README.md

```diff
@@ -204,6 +205,8 @@ python <run-file> -h
     - Start rendering after the number of episodes.
 - `--load-from <save-file-path>`
     - Load the saved models and optimizers at the beginning.
+- `--off-framestack`
```

Comment:
should remove because this flag is only for pong env.

Reply:
resolved in commit b0ed25a.